storybook/.storybook/manager-head.html

```diff
@@ -12,3 +12,7 @@
         document.body.appendChild(badge);
     });
 </script>
+<style>
+    button[id$="-docs-only"] { display: none;} 
+    div[data-item-id$="-docs-only"] { display: none;}
+</style>
```

Comment:
Do you think it's a good idea to hide the EditDialog stories? Wouldn't it be better to add a "Recommended" to the other stories?

Reply:
Yes sure, we can add a "Recommended" to the stories title, but as we talked about it, we decided to only keep the examples of the `EditDialog Component `in the docs and remove it from the stories. Hiding the story (by css styles) is the only way I could find to do that.

Reply:
This doesn't seem to work:

<img width="272" alt="Screenshot 2024-06-17 at 16 15 39" src="https://github.com/vivid-planet/comet/assets/48853629/4886581a-ac30-45f8-a40b-e55b4dd58318">